core/.dockerignore

```diff
@@ -1,3 +1,2 @@
 **/*.env
 *.env
-/tmp
```

Comment:
this was so i could build the binary in ./tmp/core-amd64 and copy it over, maybe it's better to put it into a ./bin/core-amd64 and ignore tmp as well? not sure